packages/block-library/src/cover/index.php

```diff
@@ -18,7 +18,7 @@ function render_block_core_cover( $attributes, $content ) {
 		return $content;
 	}
 
-	if ( ! ( $attributes['hasParallax'] || $attributes['isRepeated'] ) ) {
+	if ( ! ( $attributes['hasParallax'] ) ) {
```

Comment:
Hi, @amustaque97

This code should only run when the Cover block is set to use the featured image. I think we might have a different problem here.

Reply:
I think @Mamaduka is right here. Also this check seems weird:
> if ( ! ( $attributes['hasParallax'] || $attributes['isRepeated'] ) ) {

It seems for example we conditionally set `focalPoint` which doesn't seem right. If we wanted to conditionally add things we should also do it in the editor by hiding controls etc..

--cc @glendaviesnz @draganescu

Reply:
thanks for the ping - will leave it to @draganescu to comment as I haven't looked at the server rendering side of the featured image support to-date.